internal/collector/otelcol.tmpl

```diff
@@ -28,6 +28,15 @@ receivers:
   nginx/{{- .InstanceID -}}:
     endpoint: "{{- .StubStatus -}}"
     collection_interval: 10s
+    {{- if eq (len .AccessLogs) 1 }}
+    {{- range .AccessLogs }}
+    access_log_format: "{{- .LogFormat -}}"
+    operators:
+      - type: file_input
+        include:
+          - "{{- .FilePath -}}"
+    {{- end }}
+    {{- end }}
```

Comment:
So this is limited to 1 access log for the moment because this NGINX OSS receiver was originally written with the assumption that we'd use the out-of-the-box `file_input` operator (see [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/file_input.md)), which consumes either one log file or a list of log files and then pushes them up to a `stanza` pipeline (see [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/stanza)).

The issue with the above approach is that we cannot pass a log format for each access log file, meaning we the log entries from ALL access log files to the same pipeline and only apply the log format at the other end of the pipeline. At that point we have no way of knowing which entry should use which log format.

So I think we will need to write our own modified `file_input` operator that accepts a log format to circumvent the this problem.